FordLabs/PeopleMover
# Swap out font awesome icons for Material Design Icons

## Issue
Resolves #461

## What was done
- [x] Replace almost all icons in PeopleMover with icons from the Material Designs plugin

## How to test
1. Check all icons on space dashboard
2. Check all icons on a space
3. Check all icons on modals (My Tags, My Roles, Product, Person, etc)

Icons URL: https://material.io/resources/icons/?style=baseline

### FOR DEVS
Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/

Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/

### FOR APPROVAL
Approval URL: https://approvalbranchui.apps.pd01i.edc1.cf.ford.com/

Approval Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20ApprovalBranch/
